experimental/generation/generator/packages/cli/README.md

```diff
@@ -17,31 +17,68 @@ Generate Bot Framework Adaptive Dialogs from JSON schema.
 # Commands
 
 <!-- commands -->
+* [`bf dialog:generate SCHEMA`](#bf-dialoggenerate-schema)
+* [`bf dialog:generate:swagger PATH`](#bf-dialoggenerateswagger-path)
 
-- [bf dialog:generate](#bf-dialoggenerate)
+## `bf dialog:generate SCHEMA`
 
-## `bf dialog:generate`
+[PREVIEW] Generate localized .lu, .lg, .qna and .dialog assets to define a bot based on a schema using templates.
 
 ```
 USAGE
+  $ bf dialog:generate SCHEMA
 
-`bf dialog:generate`
+ARGUMENTS
+  SCHEMA  JSON Schema .schema file used to drive generation.
 
 OPTIONS
+  -f, --force                Force overwriting generated files.
+  -h, --help                 show CLI help
```

Comment:
Show CLI help

Reply:
This is part of the CLI which I don't control.